File: ecommerc_dotnet/dto/Response/DeliveryAnalysDto.cs
namespace ecommerc_dotnet.dto.Response;

public class DeliveryAnalysDto
{
    public decimal? dayFee { get; set; } = null;
    public decimal? weekFee { get; set; } = null;
    public decimal? monthFee { get; set; } = null;
    public int? dayOrder { get; set; } = null;
    public int? weekOrder { get; set; } = null;
}

File: ecommerce-delivery-man/app/src/main/java/com/example/e_commerc_delivery_man/dto/response/DeliveryAnalysDto.kt
package com.example.e_commerc_delivery_man.dto.response

import kotlinx.serialization.Serializable

@Serializable()
data class DeliveryAnalysDto(
    val dayFee: Double? = null,
    val weekFee: Double? = null,
    val monthFee: Double? = null,
    val dayOrder: Int? = null,
    val weekOrder: Int? = null
)
